.github/workflows/test.yml

```diff
@@ -230,6 +230,52 @@ jobs:
         # which pixi should be absolute
         which pixi | grep -q "^/"
 
+  existing-pixi-bin:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Move pixi.toml
+        run: mv test/default/* .
+      - name: Download pixi binary
```

Comment:
Would be cool to also test this on windows. You can do a custom step with `if: ${{ matrix.os != 'windows-latest' }}`

Reply:
Done